lib/my_app.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter/material.dart';
+import 'package:module_three_second/screens.dart';
+
+class MyApp extends StatelessWidget {
+  const MyApp({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      debugShowCheckedModeBanner: false,
```

Comment:
Looks good, you investigated this approach of navigation - now is the next step - try to use `MaterialApp.onGenerateRoute` to handle navigation - it is a more production-used variant.